zenguard/zenguard.py

```diff
@@ -87,6 +80,22 @@ def detect(self, detectors: list[Detector], prompt: str):
             return {"error": str(e)}
 
         return response.json()
+    
+    def detect_and_send_prompt(self, detectors: list[Detector], prompt: str):
+        try:
+            detected_response = self.detect(detectors, prompt)
```

Comment:
what if there is a prompt injection attempt? we should not even go into the sending response